packages/libp2p/src/connection-manager/index.ts

```diff
@@ -216,8 +218,8 @@ export class DefaultConnectionManager implements ConnectionManager, Startable {
     this.onDisconnect = this.onDisconnect.bind(this)
 
     // allow/deny lists
-    this.allow = (init.allow ?? []).map(ma => multiaddr(ma))
-    this.deny = (init.deny ?? []).map(ma => multiaddr(ma))
+    this.allow = init.allow != null ? this.parseIpNetList(init.allow) : []
+    this.deny = init.deny != null ? this.parseIpNetList(init.deny) : []
```

Comment:
The functionality from `this.parseIpNetList` is encapsulated in `multiaddrToIpNet` so let's just use that:

```suggestion
    this.allow = (init.allow ?? []).map(str => multiaddrToIpNet(str))
    this.deny = (init.allow ?? []).map(str => multiaddrToIpNet(str))
```

Reply:
Shouldn't line 222 be  this.deny = (init.deny ?? []).map(str => multiaddrToIpNet(str)) 
not init.allow ?

Reply:
Yes, good spot.